woostarter/theme.json

```diff
@@ -339,15 +339,44 @@
 					"lineHeight": "1.18"
 				}
 			},
+			"woocommerce/accordion-item": {
+				"border": {
+					"bottom": {
+						"color": "var(--wp--preset--color--theme-6)",
+						"style": "solid",
+						"width": "1px"
+					},
+					"color": null,
+					"style": null,
+					"width": null
```

Comment:
Are these set to `null` to override the accordion item's default styles?

Reply:
I think CBT did this for me, I'll check if it's needed or not

Reply:
Seems like they could be deleted, done! Thanks for the review